discover/action.yml

```diff
@@ -34,9 +34,10 @@ runs:
 
 
     - name: Install Nix
-      uses: cachix/install-nix-action@v18
+      uses: cachix/install-nix-action@pull/150/head
       if: startsWith(runner.name, 'discovery') == false
       with:
+        install_options: --no-daemon
```

Comment:
Works by eliding multi-user install. `std-action-runner` needs to potentially copy this. cc @nrdxp